engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -123,36 +121,21 @@ public List<User> getUsers(String accountId, boolean adminsOnly) {
         Timer.Sample getUsersTotalTimer = Timer.start(meterRegistry);
 
         List<User> users;
-        if (retrieveUsersFromIt) {
-            List<ITUserResponse> usersPaging;
-            List<ITUserResponse> usersTotal = new ArrayList<>();
+        List<ITUserResponse> usersPaging;
+        List<ITUserResponse> usersTotal = new ArrayList<>();
 
-            int firstResult = 0;
+        int firstResult = 0;
 
-            do {
-                Timer.Sample getUsersPageTimer = Timer.start(meterRegistry);
-
-                ITUserRequest request = new ITUserRequest(accountId, adminsOnly, firstResult, maxResultsPerPage);
-                usersPaging = retryOnItError(() -> itUserService.getUsers(request));
-                usersTotal.addAll(usersPaging);
+        do {
+            ITUserRequest request = new ITUserRequest(accountId, adminsOnly, firstResult, maxResultsPerPage);
+            usersPaging = retryOnItError(() -> itUserService.getUsers(request));
+            usersTotal.addAll(usersPaging);
 
-                firstResult += maxResultsPerPage;
+            firstResult += maxResultsPerPage;
+        } while (usersPaging.size() == maxResultsPerPage);
 
-                getUsersPageTimer.stop(meterRegistry.timer("rbac.get-users.page", "accountId", accountId));
```

Comment:
Is this no longer needed?

Reply:
Thought nobody would see any advantages in having it. If you would like to keep it, then I can add it again :)